packages/map-template/package.json

```diff
@@ -40,14 +40,15 @@
     "vite-plugin-eslint": "^1.8.1",
     "vite-plugin-favicon2": "^1.1.5",
     "vite-plugin-svgr": "^3.2.0",
-    "web-vitals": "^2.1.4"
+    "web-vitals": "^2.1.4", 
+    "jest": "^29.7.0"
```

Comment:
Let's keep them sorted alphabetically 😁

Reply:
:404annesurprisedemojinotfound:

Reply:
I did not know we sorted them alphabetically until now 😂

Reply:
I guess that's also something for the contribution guide 🙃

Reply:
Done ✅